README.md

```diff
@@ -250,6 +253,30 @@ public class HelloWorldActivity extends TiActivity<HelloWorldPresenter, HelloWor
 
 `LoggingInterceptor` is available as module and logs all calls to the view.
 
+### Kotlin
+
+Using Kotlin these days is a no brainer.
```

Comment:
Consider adding a part that the user has to include `net.grandcentrix.thirtyinch:thirtyinch-kotlin`